Sources/ImageCache.swift

```diff
@@ -226,6 +226,50 @@ open class ImageCache {
             callHandlerInMainQueue()
         }
     }
+
+    // MARK: - Add file
+
+    /**
+     Adds an existing image file to cache. It will be saved to disk only. It is a sync operation, since (URLSessionDownloadDelegate.urlSession(:downloadTask:didFinishDownloadingTo:) requires that the move/sync be done synchronously.)
+
+     Will MOVE the original file instead of coping the file if 'moveFile' is true.
+
+     Returns the destination URL of the copied or moved file.
+
+      */
+
+    open func addFile(_ originalFileURL: URL,
+                       moveFile: Bool,
+                       forKey key: String,
+                       processorIdentifier identifier: String = "") -> URL
```

Comment:
Should maybe make URL optional so it's only returned if the 'copy' or 'move'  was successful.